src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -90,6 +90,9 @@ private boolean isDefault( final String attributeValue ) {
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
+		if ( attributeValue.equals( "0px" ) ) {
```

Comment:
0px is not always default: https://www.w3schools.com/cssref/css_default_values.asp

Reply:
Accepted. Let's improve that in a better way ... but for a first approximation, it is good enough.